.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuotes": true
```

Comment:
Ideally, these sort of linting changes would be a separate PR so as not to create noise for the changes you actually want to review. The added benefit is that the PR ends up being many fewer lines of code.